cmd/ttn-lw-cli/commands/end_devices.go

```diff
@@ -297,9 +298,13 @@ var (
 			if err != nil {
 				return err
 			}
+
 			if err := macVersion.UnmarshalText([]byte(s)); err != nil {
 				return err
 			}
+			if err := macVersion.Validate(); err != nil {
+				return fmt.Errorf("LoRaWAN version is invalid: %s", err)
```

Comment:
1. The error message isn't really helpful in the CLI (because it renders the min/max enum value as numeric, and the CLI only works with string representations).
2. We try to avoid `fmt.Errorf`, better to define `errInvalidMACVersion`/`errInvalidPHYVersion` and return those

Reply:
This is the current output:
```
error:cmd/ttn-lw-cli/commands:mac_version (LoRaWAN MAC version is invalid)
--- error:pkg/ttnpb:field_bound (`MACVersion` should be between `1(1.0.0)` and `5(1.0.3)`)
    value=unknown
    lorawan_field=MACVersion
    min=1(1.0.0)
    max=5(1.0.3)
exit status 255
```

Reply:
I also don't see how this is helpful. This would imply that 1.1 wouldn't be allowed.